.travis.yml

```diff
@@ -16,8 +16,15 @@ before_script:
   - yamllint --version
 
 script:
-  - for dir in *; do test -d "$dir" && echo "$dir" && ansible-lint "$dir"; done
-  - for dir in *; do test -d "$dir" && echo "$dir" && yamllint -c ./.testenv/yamllint.yml "$dir"; done
-  - for file in */tasks/*; do egrep -H '^- name:.*[[:upper:]]' "$file" && exit 1; done
+  - for dir in *; do
+      test -d "$dir" &&
+      echo "$dir" &&
+      ansible-lint -R -r .testenv/lint-rules/ "$dir";
+    done
+  - for dir in *; do
+      test -d "$dir" &&
+      echo "$dir" &&
+      yamllint -c ./.testenv/yamllint.yml "$dir";
```

Comment:
`./.` and above you use `.testenv`, can this be aligned?

Reply:
done